jicofo/rootfs/defaults/jicofo.conf

```diff
@@ -6,6 +6,7 @@
 {{ $JICOFO_AUTH_LIFETIME := .Env.JICOFO_AUTH_LIFETIME | default "24 hours" -}}
 {{ $ENABLE_SCTP := .Env.ENABLE_SCTP | default "1" | toBool -}}
 {{ $ENABLE_RECORDING := .Env.ENABLE_RECORDING | default "0" | toBool -}}
+{{ $ENABLE_TRANSCRIPTIONS := .Env.ENABLE_TRANSCRIPTIONS | default "0" | toBool -}}
```

Comment:
I believe this is being moved for alphabetizing (which I approve of), but also duplicates an existing entry for this variable further down in the file.  I suggest remove line 16 below if we're adding this one here.